src/sentry/eventstream/kafka/backend.py

```diff
@@ -125,6 +125,8 @@ def insert(
                 {"project_id": event.project_id, "message_type": message_type},
             )
 
+        kwargs["message_type"] = message_type
+
```

Comment:
Where do we pass this to Kafka?

Reply:
might have accidentally removed it. one sec

Reply:
in the insert call below? or am I mistaken?

Reply:
This is the insert call we make here: https://github.com/getsentry/sentry/blob/6b4bbc70d51626614b2909656750b322b5b1d5a2/src/sentry/eventstream/snuba.py#L112-L123

The kwargs aren't directly used for anything, so `message_type` is just thrown into the void. It is used in the Snuba backend, but since we're passing a message to a consumer, `message_type` will need to be included as part of the headers, since that's what the consumer reads.